pose_thumbnails/core.py

```diff
@@ -740,21 +755,23 @@ def poll(cls, context):
 
     def draw(self, context):
         addon_prefs = prefs.for_addon(context)
-        wm = context.window_manager
         obj = context.object
         poselib = obj.pose_library
         layout = self.layout
         col = layout.column(align=True)
-
+        if obj.name.startswith(addon_prefs.optional_name_prefix):
+            char = obj.name[len(addon_prefs.optional_name_prefix):]
+        else:
+            char = obj.name
         col.prop(
             obj,
             'pose_lib_for_char',
             text='Libraries for {char}'.format(
-                char=character_name(obj.name, context)),
+                char=character_name(char, context)),
         )
         col.separator()
         if poselib and poselib.pose_markers:
-            pose_thumbnail_options = wm.pose_thumbnails.options
+            pose_thumbnail_options = context.window_manager.pose_thumbnails.options
```

Comment:
Style change only. There the two draw functions do nearly the exact same thing (except the properties panel one one has the edit thumbnails view). So I just made the code look variables and line order more between the two panels, and since wm is only used once (and the place it's used still fit within 80 characters) I chose to remove it in this panel, as opposed to adding the `wm` var to the other panel.

Quite indifferent to do the reverse, and only made the chance as I was looking at the function to add the Show All update function.

Reply:
I'm perfectly fine with this. I was just wondering why you changed it. :)